backend/app/views/spree/admin/users/_form.html.erb

```diff
@@ -58,21 +58,4 @@
     <% end %>
 
   </div>
-
-  <div data-hook="admin_user_form_password_fields" class="col-6">
```

Comment:
I'm not sure if we should delete this div with the form fields or not. There maybe stores using this data-hook so I'm thinking that perhaps we should leave it in for backwards compatibility. Although I'm also annoyed the data-hook would have a bad name.

Reply:
If I got it correctly, I think this `div` should go into the `if` statement. I think it will still be evaluated even if the `if` statement is `false`, since Deface should do its magic before the erb is rendered.

Reply:
My thought here was that someone may have a deface customization that's adding non-password related fields here simply because it would be a convenient place to add more user fields. I'm fine putting the div into the conditional statement as well, but I think that may cause customizations to be hidden unintentionally if people are using this data hook for non-password stuff.

Reply:
I think that if someone is adding some non-password related fields into the password dedicated div it's ok to make this change, we can't handle every scenario. The problem is if someone is using this hook in the `insert_after` or `insert_before` deface action, which is legit and the added parts will be hidden at that point. :( 

That said, your solution is probably the safest one in terms of backward compatibility.